dashboard/app/controllers/application_controller.rb

```diff
@@ -144,6 +147,13 @@ def milestone_response(options)
       response[:share_failure] = options[:share_failure]
     end
 
+    if HintViewRequest.enabled?
```

Comment:
oh, right you are. I grabbed the wrong chunk from https://github.com/code-dot-org/code-dot-org/pull/4753/files#diff-8da26fef534bb3b421a98e21f07ff19bR163